cmd/ingester/app/flags.go

```diff
@@ -49,6 +49,8 @@ const (
 	SuffixParallelism = ".parallelism"
 	// SuffixHTTPPort is a suffix for the HTTP port
 	SuffixHTTPPort = ".http-port"
+	// SuffixKafkaProtocolVersion Kafka protocol version - must be supported by kafka server
+	SuffixKafkaProtocolVersion = ".protocol-version"
```

Comment:
this should be grouped with Kafka-related field above, above `.encoding` 

Probably doesn't need Kafka in the name